client/my-sites/themes/themes-selection.jsx

```diff
@@ -66,14 +66,16 @@ const ThemesSelection = React.createClass( {
 		}
 	},
 
-	recordSearchResultsClick( theme, resultsRank ) {
+	recordSearchResultsClick( theme, resultsRank, action ) {
 		const { query, themes } = this.props;
 		analytics.tracks.recordEvent( 'calypso_themeshowcase_theme_click', {
 			search_term: query.search,
 			theme: theme.id,
 			results_rank: resultsRank + 1,
 			results: themes.map( property( 'id' ) ).join(),
-			page_number: query.page
+			page_number: query.page,
+			theme_on_page: parseInt( ( resultsRank + 1 ) / query.number ),
+			action: snakeCase( action )
```

Comment:
I did the convert because at this level I only have access to labels and they have spaces, `&` character, one or two words. It would just look messy and because that less readable + it will look like the analytics tag. In general you are right not very important, but I am for keeping it.